1-exercises/A-this/exercise-2.js

```diff
@@ -11,7 +11,9 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function (newAddress) {
+      return (this.currentAddress = newAddress);  
```

Comment:
Good Job Mohemed, have you try without return? what will happened if you don't put return?